cmd/main.go

```diff
@@ -1,5 +1,38 @@
 package main
 
+import (
+	"net/http"
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"github.com/tech-pool/takemetotheshop-api/api/registration"
+)
+
+var port string
+
+func init() {
+	port = os.Getenv("PORT")
```

Comment:
Should we also have a default PORT when nothing is passed? Just so that it's easier for local development? Also is there something like dotenv in golang? For easier default environment variable setting during local development?